docs/editors/new-editor.md

```diff
@@ -541,6 +541,73 @@ export interface MetalsInputBoxResult {
 }
 ```
 
+### `metals/pickInput`
```

Comment:
Would it make sense to name this `metals/quickPick`? In general I try to follow the same naming convention as in the VS Code APIs

Reply:
Yeah, I was not sure about the name, so if you think, this one's better, okay then. Renamed.